.github/actions/prepare-frontend/action.yml

```diff
@@ -7,13 +7,21 @@ runs:
       with:
         node-version: 14.x
         cache: 'yarn'
+    - name: Check to see if dependencies should be cached
+      if: ${{ contains(github.event.head_commit.message, '[ci nocache]') }}
+      run: |
+        echo "Commit message includes [ci nocache]; dependencies will NOT be cached"
```

Comment:
Here the message should be something around `Commit message includes [ci nocache]; dependencies cache will be cleaned` to properly say what is going to happen, right ?

Reply:
See the comment on the one, the log correctly informs _what_ will happen (not _how_ it is being implemented).